config/operator-bundle/patches/admissionreviewversions_in_webhook.yaml

```diff
@@ -0,0 +1,17 @@
+# This adds an admissionReviewVersions field to the validation webhook
+# items.  It's not exactly valid, since it's a v1beta1
```

Comment:
Should we file a bug for the operator bundle folks?

Reply:
Good call - I'm doing that now.

Reply:
Actually, while I was writing up a bug for them I realised there was nothing stopping us from switching to v1 ValidationWebhookConfiguration, which would make this less weird. So I did that instead, although it required patching the file to remove v1 from admissionReviewVersions (since those are not supported by controller-runtime).

Reply:
I realised the patches aren't applied when running the tests, so any test relying on a webhook failed. Unfortunately there's no option in the kubebuilder webhook directive to manage the versions directly. 
Actually I could just do it directly in the manifests make target like the `preserveUnknownFields` fix... I'll land this one as it is and do that subsequently.